Sources/POI/OAPOI.h

```diff
@@ -56,34 +57,28 @@
 
 @end
 
-@interface OAPOI : NSObject
+@interface OAPOI : OAMapObject
 
-@property (nonatomic) unsigned long long obfId;
-@property (nonatomic) NSString *name;
 @property (nonatomic) OAPOIType *type;
 @property (nonatomic) NSString *subType;
-@property (nonatomic) NSString *nameLocalized;
 @property (nonatomic, assign) BOOL hasOpeningHours;
 @property (nonatomic) NSString *openingHours;
 @property (nonatomic) NSString *desc;
 @property (nonatomic) BOOL isPlace;
 @property (nonatomic) NSString *buildingNumber;
 
-@property (nonatomic, assign) double latitude;
-@property (nonatomic, assign) double longitude;
 @property (nonatomic, assign) double distanceMeters;
 @property (nonatomic) NSString *distance;
 @property (nonatomic, assign) double direction;
 
 @property (nonatomic) NSDictionary *values;
-@property (nonatomic) NSDictionary *localizedNames;
 @property (nonatomic) NSDictionary *localizedContent;
 
 @property (nonatomic) OAPOIRoutePoint *routePoint;
 @property (nonatomic) NSString *mapIconName;
 @property (nonatomic) NSString *cityName;
 
-@property (nonatomic, assign) BOOL isRenderedObject;
+@property (nonatomic) BOOL isRenderedObject;
```

Comment:
Not clear why `isRenderedObject` needed if `OARenderedObject` introduced already